subsys/net/openthread/rpc/common/ot_rpc_common.c

```diff
@@ -540,3 +540,33 @@ bool ot_rpc_decode_border_router_config(struct nrf_rpc_cbor_ctx *ctx, otBorderRo
 
 	return true;
 }
+
+bool ot_rpc_encode_sockaddr(struct nrf_rpc_cbor_ctx *ctx, const otSockAddr *sockaddr)
+{
+	struct zcbor_string bstr = { .value = &sockaddr->mAddress, .len = OT_IP6_ADDRESS_SIZE };
```

Comment:
You can use `zcbor_bstr_encode_ptr` to avoid creating this.

Reply:
Isn't better to use for buffer encoding decoding already implemented functionality?
[nrf_rpc_encode_str](https://github.com/nrfconnect/sdk-nrf/blob/main/include/nrf_rpc/nrf_rpc_serialize.h#L123-L129)
https://github.com/nrfconnect/sdk-nrf/blob/main/include/nrf_rpc/nrf_rpc_serialize.h#L241-L258

Reply:
Fixed